project/Build.scala

```diff
@@ -135,9 +137,69 @@ object Build extends sbt.Build {
             .find(_.endsWith(java.io.File.separator + "rt.jar")).get
         }
 
-        file(rtJar) -> url("http://docs.oracle.com/javase/8/docs/api/")
-      }
+        file(rtJar) -> url(javaDocBaseURL)
+      },
+
+
+      /* Patch the ScalaDoc we generate.
+       *
+       *  After executing the normal doc command, copy everything to the
+       *  `patched-api` directory (same internal directory structure) while
+       *  patching the following:
+       *
+       *  - Append `additional-doc-styles.css` to `lib/template.css`
+       *  - Fix external links to the JavaDoc. I.e. change
+       *    `${javaDocBaseURL}index.html#java.lang.String` to
+       *    `${javaDocBaseURL}index.html?java/lang/String.html`
```

Comment:
What if we link to a *member* of a class? Or to a "static" method, which would be in a companion object in Scala but as static in Java?